plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -232,11 +228,7 @@ export class EventsProcessor {
         propertiesOnce: Properties,
         timestamp: DateTime
     ): Promise<void> {
-        if (this.isNewPersonPropertiesUpdateEnabled(teamId)) {
-            await updatePersonProperties(this.db, teamId, distinctId, properties, propertiesOnce, timestamp)
-        } else {
-            await this.updatePersonPropertiesDeprecated(teamId, distinctId, properties, propertiesOnce)
-        }
+        await this.updatePersonPropertiesDeprecated(teamId, distinctId, properties, propertiesOnce)
```

Comment:
Are we going to keep the name as "deprecated" given we're deleting the old code?

Reply:
I'm happy to merge, but would like to change this soon. If someone other than you or me looks at the codebase they will be very confused as to what the non-deprecated method is. 

Let's at least add a comment. I also don't get the point about the tests. Now that they all pass you shouldn't need this anymore.

Reply:
yes, the tests part was while they hadn't all passed, like you guessed.